README.md

```diff
@@ -92,15 +95,15 @@ Run `pdig` with the path (and arguments, if any) of the process you want to trac
 
     pdig [-a] curl https://example.com/
 
-The `-a` option enables the full filter, which provides a richer set of instrumented system calls. You probably want to use this option with sysdig, but not with falco.
+The `-a` option enables the full filter, which provides a richer set of instrumented system calls. You probably don't want to use this option  with Falco for performance reasons.
```

Comment:
`sysdig` is not Sysdig Inc. I don't believe removing information useful for OSS Sysdig users achieves anything. Same goes for the next chunk.